### ISSUE TITLE
 'TRADFRI bulb E27 WS opal 980lm' not recognised as light

### ISSUE THREAD

**Issue by svd98** [ggravlingen/pytradfri#165]
### Where are you using pytradfri (eg stand-alone, Home Assistant etc)
from within PyCharm

### Version of pytradfri
latest one pulled fro, github (not sure where to find the version)

### Expected behaviour
The light should be detected

### Actual behaviour
The light is not detected

The following code snippet
    devices_command = gateway.get_devices()
    devices_commands = api(devices_command)
    devices = api(devices_commands)

    for dev in devices:
        print(dev.raw)
    exit()

returns
```
{  
   '15009':[  
      {  
         '9003':0
      }
   ],
   '9002':1526150447,
   '9054':0,
   '3':{  
      '9':74,
      '2':'',
      '6':3,
      '1':'TRADFRI remote control',
      '0':'IKEA of Sweden',
      '3':'1.2.214'
   },
   '9001':'TRADFRI remote control',
   '9003':65536,
   '9019':1,
   '5750':0,
   '9020':1526154601
}{  
   '15009':[  
      {  
         '5717':0,
         '5851':254,
         '9003':0,
         '5709':30138,
         '5706':'f1e0b5',
         '5711':370,
         '5850':1,
         '5710':26909
      }
   ],
   '9002':1526154589,
   '9054':0,
   '3':{  
      '6':1,
      '2':'',
      '0':'IKEA of Sweden',
      '3':'1.2.217',
      '1':'TRADFRI bulb E27 WS opal 980lm'
   },
   '9001':'TRADFRI bulb E27 WS opal 980lm',
   '9003':65538,
   '9019':1,
   '5750':0,
}
```

I just updated the bulb from firmware version 1.1.1 to 1.2.217 and I hoped this would fix the problem but alas.  Also rebooting the hub did not help

Any Ideas?

**Comment by ggravlingen** [comment]
It's a known bug with the gateway and/or the bulb. Please have a look here for some possible solutions:
https://github.com/ggravlingen/pytradfri/issues/135

**Comment by svd98** [comment]
Thanks for your quick reply (and the work you put into this). I got it somehow working :D 

So first I did as discussed above the firmware update, but still it was found as a device.
This morning I did a full reset of the bulb (6 times on/off) and reconnected it through the app and now it was found.

Don't know which step was crucial for succes or if it was just a lucky shot, but it also may be intresting information for the #135 topic.
